### PR TITLE
Renaming functions and variables

### LINKED ISSUE #47: Rename camel-case functions
Rename the functions to be lowercase, with words separated by underscores as necessary to improve readability

### LINKED ISSUE #48: Deleting unnecessary comments
Deleting unnecessary comments from code files

### PR DESCRIPTION
**Pull request by Shalevro2**
Updating article.py & test_article.py- renaming functions and variables Renaming functions & variables that are camel-case

Updating user.py- deleting unnecessary comments

Resolve #47 
Resolve #48

### CODE REVIEW

File: projboard/tests/test_article.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def create_articles(self, create_user, create_subjects):
 
     @pytest.mark.skip
     @pytest.mark.django_db
-    def test_objectCreation(self, create_articles):
+    def test_object_creation(self, create_articles):
```
Comment (Anas-mhameed) [inline projboard/tests/test_article.py]: You can remove this function , we check this functionality in migration files.

**Review by Anas-mhameed — CHANGES_REQUESTED** [review]
Remove all @pytest.mark.skip in test_article.py

File: projboard/tests/test_article.py (hunk 1 above)
Comment (Shalevro2) [reply projboard/tests/test_article.py]: I don't think this change need to be at this PR.
We can create a new issue for not overload on this PR.
WDYT?

**Review by yaelnoyman1 — APPROVED** [review]

**Comment by Shalevro2** [comment]
> Remove all @pytest.mark.skip in test_article.py

Done at [b28a322](https://github.com/redhat-beyond/Smarticle/pull/50/commits/b28a3229b75a790d489725099000e5436a78db90)

File: projboard/tests/test_article.py (hunk 1 above)
Comment (Anas-mhameed) [reply projboard/tests/test_article.py]: It's just a small fix , i don't think we really should open a new assue for that , WDYT ?

**Review by Anas-mhameed — APPROVED** [review]
LGTM

**Review by rawadabu — APPROVED** [review]
LGTM

File: projboard/tests/test_article.py (hunk 1 above)
Comment (Yarboa) [reply projboard/tests/test_article.py]: +1 @Anas-mhameed 
Can be deleted. 
Could be done in different PR also

**Review by Yarboa — APPROVED** [review]

**Review by NitzanMordhai — APPROVED** [review]

**Review by finziamit — APPROVED** [review]
LGTM

**Comment by avi-biton** [comment]
LGTM, you can remove `test_object_creation` in this MR

File: projboard/tests/test_article.py (hunk 1 above)
Comment (Shalevro2) [reply projboard/tests/test_article.py]: Done

**Review by Yarboa — APPROVED** [review]

**Pull request merged**